### PR TITLE
DAT-72188 table design changes

### PR DESCRIPTION
**Pull request by itaydafna**
## Description
Write here a short description

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Are you adding a new package?
- [ ] yes
- [x] no

### CODE REVIEW

File: src/components/Grid/Body.tsx [hunk 1]
```diff
@@ -65,6 +65,7 @@ const rowRenderer = (props: RowRendererProps) => {
                 rowIndex: index,
                 key: header.dataKey,
                 value: get(header.dataKey, data[index]),
+                data,
```
Comment (itaydafna) [inline src/components/Grid/Body.tsx:68]: Upon search - the passed `rowIndex` is of the filtered data - so it doesn't allow searching the unfiltered data - hence I'm passing the filtered data as an additional prop.

**Review by itaydafna — COMMENTED** [review]
Added a comment that explains this change.

**Review by crazypixel — APPROVED** [review]

**Pull request merged**